test/gst-msdk/encode/av1.py

```diff
@@ -0,0 +1,58 @@
+###
+### Copyright (C) 2019-2021 Intel Corporation
+###
+### SPDX-License-Identifier: BSD-3-Clause
+###
+
+from ....lib import *
+from ....lib.gstreamer.msdk.util import *
+from ....lib.gstreamer.msdk.encoder import EncoderTest
+
+spec = load_test_spec("av1", "encode", "8bit")
+
+@slash.requires(*have_gst_element("msdkav1enc"))
+@slash.requires(*have_gst_element("msdkav1dec"))
+class AV1EncoderBaseTest(EncoderTest):
+  def before(self):
+    super().before()
+    vars(self).update(
+      codec         = "av1-8",
+      gstencoder    = "msdkav1enc",
+      gstdecoder    = "msdkav1dec",
+      gstmediatype  = "video/x-av1",
+      gstmuxer      = "matroskamux",
+      gstdemuxer    = "matroskademux",      
```

Comment:
Do we need `gstparser = "av1parse"`... or, maybe `gstparser = "av1parse ! ivfparse"`?

Reply:
and possibly `"ivfparse"` for both mux/demux?

Reply:
Since file extension is `.ivf`, how about we use the following instead:
```python
gstmuxer    = "ivfparse",
gstdemuxer  = "ivfparse",
gstparser   = "av1parse",
```